CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ExaDG
+We are happy that you wish to contribute to ExaDG. 
+Here you will find some useful information you need before you start developing. 
+
+## Getting started
+---
+Make sure you have read our [README.md](https://github.com/exadg/exadg/blob/057d6c8ea7c6cb273d98ae4b5edb197cf03aaffe/README.md) to understand what ExaDG is and how we want to maintain the project.
```

Comment:
I would formulate this a bit less offensive, e.g. "Please read our readme for general information on ExaDG and how we ..."